pingu/corpus/assets/label.py

```diff
@@ -33,6 +35,26 @@ def __lt__(self, other):
     def __repr__(self) -> str:
         return 'Label({}, {}, {})'.format(self.value, self.start, self.end)
 
+    def read_samples(self, sr=None):
+        """
+        Read the samples of the utterance.
+
+        Args:
+            sr (int): If None uses the sampling rate given by the file, otherwise resamples to the given sampling rate.
```

Comment:
What happens if sr is bigger than the sampling rate of the file? Exception?

Reply:
Librosa returns an upsampled signal.